### ISSUE TITLE
docker container write logs to the host's /var/log/kern.log

### ISSUE THREAD

**Issue by lyc0221** [martin-helmich/prometheus-nginxlog-exporter#129]
**Describe the issue**
prometheus-nginxlog-exporter container write logs to the host's /var/log/kern.log
Why ?

**Configuration file** (remove section, if not applicable):
config.hcl
```
listen {
  port = 4040
}

consul {
  enable = false
  address = "localhost:8500"
  datacenter = "dc1"
  scheme = "http"
  token = ""
  service {
    id = "nginx-exporter"
    name = "nginx-exporter"
    tags = ["foo", "bar"]
  }
}

namespace "nginx" {
  format = "$host $remote_addr $remote_user [$time_local] $uri \"$request\" $request_length $status $bytes_sent $body_bytes_sent $request_time \"$http_referer\" \"$http_user_agent\" $content_type $upstream_cache_status $upstream_response_time $upstream_addr"
  source {
    files = [
      "/var/log/nginx/access.log"
    ]
  }

  histogram_buckets = [.005, .01, .025, .05, .1, .25, .5, 1, 2.5, 5, 10]
```

**docker-compose.yml**:

```
  nginx-log-exporter:
    image: prometheus-nginxlog-exporter:v0.1.0
    command: /usr/bin/prometheus-nginxlog-exporter -config-file /etc/prometheus-nginxlog-exporter.hcl
    restart: always
    container_name: prometheus-nginxlog-exporter
    ports:
      - "9114:4040"
    volumes:
      - "/opt/log/nginx/servers.log:/var/log/nginx/access.log"
      - "./config.hcl:/etc/prometheus-nginxlog-exporter.hcl"
```

**Environment**:

- Exporter version:
- OS (e.g. from /etc/os-release): Ubuntu 16.04
- Deployment method (e.g. Docker image, deb/rpm package, self-compiled, ...): docker image

**Comment by martin-helmich** [comment]
Thanks for the report. So, what _does_ the exporter log to `/var/log/kern.log`? Also, what version of the exporter are you using? `prometheus-nginxlog-exporter:v0.1.0` doesn't look like the official image.

**Comment by lyc0221** [comment]
The exporter writes the nginx access log to /var/log/kern.log
The exporter version is 0.6.0
My image comes from [this](https://github.com/implustech/prometheus-nginx-exporter/blob/master/Dockerfile)

**Comment by lyc0221** [comment]
> The exporter writes the nginx access log to `/var/log/kern.log` and `/var/log/syslog`
> The exporter version is 0.6.0
> My image comes from [this](https://github.com/implustech/prometheus-nginx-exporter/blob/master/Dockerfile)

@martin-helmich can you help me resolve this ? Thanks

**Comment by martin-helmich** [comment]
The access logs being written to stdout was a bug, fixed in #108. The fix was included in 1.4.2 -- the image that you're using seems to be using 1.4.1.

Please also note that I cannot give support for Docker images provided by 3rd parties.

However, in general the exporter only logs to STDOUT. If these logs are forwarded to your host's kernel log, this points to a misconfiguration of your host's container engine.
